src/ophys_etl/modules/suite2p_registration/utils.py

```diff
@@ -3,6 +3,125 @@
 from typing import Tuple, List
 from scipy.ndimage.filters import median_filter
 from ophys_etl.utils.array_utils import normalize_array
+from suite2p.registration.rigid import shift_frame
+
+
+def find_movie_start_end_empty_frames(
```

Comment:
It's similar but not the same. I can rename `trim_frames` to make them a bit distinct. The ticket here creates a function to specifically address problems with some of the movies having empty frames at the start and end. In this case, the function `trim_frames` you are commenting on generically removes frames that may be anywhere in the movie rather than the start/end to create the cleanest reference image. They are doing slightly different jobs.

Reply:
Changed the name of `trim_frames` to `remove_extrema_frames` and updated to the docstring to help clarify the difference.